com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/ResourceManager.java

```diff
@@ -90,20 +91,44 @@ private void registerExtensions(Set<EPackageConfiguration> configurations) {
 		return loadResource(URI.createURI(resourceURI), rs);
 	}
 
-	public <T extends EObject> Optional<T> loadModel(URI resourceURI, ResourceSet rs, Class<T> clazz) {
+	/**
+	 * Attempts to find a file resource in a casa insensitive way.
+	 */
+	private static String findResource(String filename)	{
```

Comment:
Why not return `File`?  And then we won't have to be concerned with how to compose path names.